core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -154,7 +154,7 @@ private[chisel3] object ir {
   }
 
   case class ULit(n: BigInt, w: Width) extends LitArg(n, w) {
-    def name:     String = "UInt" + width + "(0h0" + num.toString(16) + ")"
+    def name:     String = "UInt" + width + "(0h" + num.toString(16) + ")"
```

Comment:
This is the kind of thing that _could_ cause a bug. However, this looks fine. This should also cause simulations to fail if it was a problem (hopefully).

This is 💯 cleaner output, though.

Reply:
Yeah I could back this out--this change will be needed once I updated Properties to stop using FIRRTL IR and the converter, but it seems I backed out the rest of that change but not this. I do think we should still do this because it makes it consistent with what the serializer is doing (and ultimately we can call this from the serializer). I'll check for issues though of course.